units/INC0001/INC0001_unit.bp

```diff
@@ -118,17 +125,29 @@ UnitBlueprint {
             LAYER_Sub = false,
             LAYER_Water = false,
         },
+        Elevation = 5,
         MeshExtentsX = 0.9,
         MeshExtentsY = 0.55,
         MeshExtentsZ = 1.05,
         MotionType = 'RULEUMT_None',
     },
-    SelectionSizeX = 0.45,
-    SelectionSizeZ = 0.65,
-    SelectionThickness = 0.66,
-    SizeX = 0.65,
-    SizeY = 0.4,
-    SizeZ = 0.9,
+    Rotators = {
+        Stationary = false,
+        --if Stationary == true
+        -- PrimaryAngle = 45,
+        -- SecondaryAngle = -45,
+        --if Stationary == false
+        PrimaryAccel = 5,
+        PrimarySpeed = 10,
+        SecondaryAccel = -5,
+        SecondarySpeed = -20,
+    },
+    SelectionSizeX = 3,
+    SelectionSizeZ = 8,
+    SelectionThickness = 0.15,
+    SizeX = 2.5,
```

Comment:
yes it is, and it's currently not efficient https://puu.sh/tsJRd/b64cd449ed.jpg
Shift F6 will open the window where you can edit the values, Ctrl Shift Alt C will show the hitbox
if you change it you need to spawn the unit again for the changes to show